src/Components/Agenda/AgendaItemLegsFormReadOnly/AgendaItemLegsFormReadOnly.jsx

```diff
@@ -0,0 +1,142 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import FA from 'react-fontawesome';
+import InteractiveElement from 'Components/InteractiveElement';
+import { formatDate } from 'utilities';
+import { DEFAULT_TEXT } from 'Constants/SystemMessages';
+import Alert from '../../Alert';
+
+const AgendaItemLegsFormReadOnly = props => {
+  const { legs } = props;
+
+  const showOverlay = !legs.length;
+  const [rowHoverNum, setRowHoverNum] = useState();
+
+  const onHover = row => {
+    // to avoid highlighting the arrow row
+    // Note: varies by -1 from the editable version bc of the row of Xs to remove legs
+    if (row !== 7) {
+      setRowHoverNum(row);
+    }
+  };
+
+  const formatLang = (langArr = []) => langArr.map(lang => (
+    `${lang.code} ${lang.spoken_proficiency}/${lang.reading_proficiency}`
+  )).join(', ');
+
```

Comment:
```suggestion

  const getVice = (viceObj) => {
    const vice = viceObj?.emp_full_name;
    const vacancy = viceObj?.asgd_etd_ted_date && formatDate(viceObj.asgd_etd_ted_date, 'MM/YY');
    if (vice || vacancy) {
      return `${vice || ''}${(vice && vice !== 'Multiple Incumbents') ? ', ' : ''} ${vacancy || ''}`;
    }
    return '-';
  };
  
```

adds vice stuff from my pr, so it doesn't get missed

Reply:
<img width="972" alt="image" src="https://github.com/MetaPhase-Consulting/State-TalentMAP/assets/12723877/626d32f8-a496-4bda-b216-ec9a2308910d">

Reply:
no merge conflicts/clean merge, so we should be good 🦜

Reply:
but now im curious what happens if i commit your suggestion 🤔 
ima try it!

Reply:
it's beautiful!